active/PagerControl/PagerControl.md

```diff
@@ -308,25 +330,29 @@ For example, implementation details. -->
 ### UI Automation Patterns 
 
 ### Keyboarding
-
-* End users should be able to tab to focus on the control. 
+* Pager Control should be added as a navigation landmark. 
+* End users should be able to tab to put focus on the control. 
 * Left and right arrows will navigate between the items in the control. 
-* Page up and page down buttons should advance the focused index by a large amount (the amount can be configured by the developer).  
-        * An example is if the user is interacting with the number panel mode and the focus is on the page 2 button, when the user hits page down, the focus will move to the page 12 button (if page 12 exists) or the last page button. 
 * Home/escape keys should move focus from an individual item back to the entire control.
-* (Combo box mode only) If the end user is focused on the combo box, the user can hit enter to expand the drop down menu.
-* (Combo box mode only) If the end user is focused on the combo box and the drop down is expanded, the home/escape key should collapse the drop down menu and have focus remain on the combo box. 
-* (Combo box mode only) Alt + down shortcut should expand the drop down menu 
-* (Combo box mode only) Alt + up shortcut should collapse the drop down menu
-* (Number box mode only) Up and down arrows should increase or decrease the value in the number box by 1. 
+
+Accessibility for Combo Box Only
```

Comment:
Is this behavior different than the default behavior? If not, we should just refer to that definition rather than duplicating it here.

Reply:
I am going to remove this from the spec.